src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -2,13 +2,28 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
 
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
 using System.Security.AccessControl;
+using System.Security.Principal;
+using Microsoft.VisualBasic;
 using Xunit;
 
 namespace System.IO
 {
     public class FileSystemAclExtensionsTests
     {
+        #region Private members
+
+        //private static readonly Lazy<bool> s_isElevated = new Lazy<bool>(AdminHelpers.IsProcessElevated);
```

Comment:
Don't leave the commented out code in here. Create an issue to add access denied tests for directories to track.